src/Graph.elm

```diff
@@ -684,10 +652,9 @@ mapContexts f =
     fold (\ctx -> insert (f ctx)) empty
 
 
-{-| Maps over node labels, possibly changing their types. Leaves the graph
-topology intact.
+{-| Maps over node labels Leaves the graph topology intact.
```

Comment:
Note the change in API (0.19 won't allow changing record types).  Same applies for `mapEdges` below.

Reply:
I managed to fix this